src/patternfly/components/Table/examples/Table.md

```diff
@@ -134,7 +134,7 @@ These classes can be used to ensure that the table changes between the tabular a
 
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-m-grid-md`, `.pf-m-grid-lg`, `.pf-m-grid-xl`, `.pf-m-grid-2xl` | `.pf-c-table` | Changes tabular layout to responsive, grid based layout at suffixed breakpoint. |
+| `.pf-m-grid-md`, `.pf-m-grid-lg`, `.pf-m-grid-xl`, `.pf-m-grid-2xl` | `.pf-c-table` | Changes tabular layout to responsive, grid based layout at suffixed [breakpoint](/developer-resources/global-css-variables#breakpoint-variables-and-class-suffixes). |
```

Comment:
```suggestion
| `.pf-m-grid-md`, `.pf-m-grid-lg`, `.pf-m-grid-xl`, `.pf-m-grid-2xl` | `.pf-c-table` | Changes tabular layout to responsive, grid based layout at suffixed optional [breakpoint](/developer-resources/global-css-variables#breakpoint-variables-and-class-suffixes). |
```

Reply:
@mattnolting This one is listed differently than others, not using `{-on-[breakpoint]}` because there's the comment about using Javascript on the following entry without the suffixes. Do you still think "optional" makes sense there?